module/VuFind/src/VuFind/Mailer/Mailer.php

```diff
@@ -231,6 +288,15 @@ public function send($to, $from, $subject, $body, $cc = null, $replyTo = null)
             if ($replyTo) {
                 $message->addReplyTo($replyTo);
             }
+
+            $headers = $message->getHeaders();
+            if (!$headers->has('Content-Type')) {
```

Comment:
Just curious why this is necessary now -- is this something we always should have been doing for text messages, or is it only needed because of the addition of multi-part support?

Reply:
The new introduced method 'getNewBlankMessage()' does not set a Content-Type. But normally this should happen within  Zend\Mail\Message:setBody depending on body (instance of Mime\Message and at least one part with content-type set).  This external (zend / laminas) magic seemed not totally predictable, so content-type is set in Mailer class to play it safe. But I have to admit, at the moment that is unnecessary.

Reply:
If this whole block of code is only here "just in case" but you've never found a use case that actually needs it, it might actually be safer to omit it and let the external code do its magic, since without a real-world use case, the code is impossible to test, and might actually introduce problems over time. On the other hand, if there is a sequence of events that causes this to trigger, we should make sure it's covered in the unit test and perhaps commented on here just to make things more clear.

Reply:
I could not find an example, where it's useful, so I agree to delete  this peace of code. I would do that, but I am not sure about the workflow (as I mentioned the question above)?